platform/windows/os_windows.cpp

```diff
@@ -1332,7 +1332,11 @@ void OS_Windows::vprint(const char* p_format, va_list p_list, bool p_stderr) {
 	if (p_stderr)
 		fwprintf(stderr,L"%s",wbuf);
```

Comment:
How about `stderr`, doesn't it need `%S` on MinGW too?
Also, is the distinction necessary or would MSVC handle `%S` just as well?